### PR TITLE
Add open source roadmap in the readme format

### LINKED ISSUE #198: [FEATURE] Add open source roadmap in the readme format
### Description

We need to convert the graphical chart present on [cc roadmaps website] (https://www.commclassroom.org/opensource-roadmap) in the readme format

**Create Readme.md file and add the converted map there**

Eg:

```md
# Open Source

## Introduction
[Resource 1](...)
```

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by tinniaru3005**
## Fixes Issue
Closes #198

### CODE REVIEW

File: Open-Source/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Open Source
+
+## Introduction
```
Comment (siddhant-khisty) [inline Open-Source/README.md:3]: Remove the Introduction line, and make `What is Open Source` as a h2(two hashtags`

File: Open-Source/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+### What is open source
+- [Resource 1](https://www.youtube.com/watch?v=msyGybzCKRs)
+- [Resource 2](https://www.youtube.com/watch?v=7c0IrsDsNaw)
+- [Resource 3](https://opensource.com/resources/what-open-source)
+- [Resource 4](https://www.redhat.com/en/topics/open-source/what-is-open-source)
+- [Resource 5](https://en.wikipedia.org/wiki/Open-source_software)
+### Open source guide
+- [Open source guide](https://opensource.guide/)
+### What is Open Source and How Does It Work
+- [What is Open Source and How Does It Work](https://www.youtube.com/watch?v=a4ap0R_OQIE)
```
Comment (siddhant-khisty) [inline Open-Source/README.md:13]: Please group this with `What is Open Source`

**Review by siddhant-khisty — CHANGES_REQUESTED** [review]
Just a few changes to make it look better

**Comment by tinniaru3005** [comment]
Hey @siddhant-khisty, I have made the changes. Can you please confirm if that's fine now?

File: Open-Source/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+# Open Source
+
+## What is Open Source
+### What is open source
```
Comment (siddhant-khisty) [inline Open-Source/README.md:4]: remove the h3 one. It becomes repetative

**Review by siddhant-khisty — CHANGES_REQUESTED** [review]
just one change

**Review by siddhant-khisty — APPROVED** [review]
LGTM. cc @kaiwalyakoparkar

File: Open-Source/README.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+## Contributing to open source projects
+- [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/)
+- [How to Find Good First Issues in a project](https://www.youtube.com/watch?v=ZIkkPzTuOXw)
+- [How to Open Source Like a Pro](https://www.youtube.com/watch?v=MT6M_sqAuZo)
+- [How to Understand Large Codebases in Open Source](https://www.youtube.com/watch?v=kA1T0zDPtQo)
+- [How to contribute to and maintain open source projects with GitHub](https://www.oswc.is/search-projects)
+
+## Open source programs students/working professionals
+### 25+ paid open source programs and internships
```
Comment (kaiwalyakoparkar) [inline Open-Source/README.md:38]: Add blank line above

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

**Comment by tinniaru3005** [comment]
Hey @kaiwalyakoparkar, I have made the changes. Can you please look into it?

**Pull request merged**